### DISCUSSION TITLE
pageContext.urlParsed.hash isn't available on the server-side (HTTP requests don't include the URL hash by design)

### DISCUSSION THREAD

**Discussion by ashoktamang** [vikejs/vike#1114] [Discussion]
I understand that hash and hashOriginal is only available in the browser. I am wondering how can I make this warning go away and make the plugin happy?

I am passing 'urlParsed' to `passToClient` array and trying to use the `search` (query params) in *.page.client.tsx files (client pages).

**Comment by brillout** [comment]
> I am passing 'urlParsed' to `passToClient`

This was indeed an issue which was fixed a little while ago. Try again with the latest version and let me know whether you still get the warning.
